data/coalition/coalition missions.txt

```diff
@@ -2152,3 +2152,185 @@ mission "Coalition: Council of Ahr: On Complete"
+		random < 25
+	source
+		government "Coalition"
+		near "5 Axis" 1 3
+	destination "Market of Gupta"
+	on offer
+		conversation
+			`You notice a Kimek anxiously glancing at the docks from the front of an office building. Once he notices you, he scurries closer. "Say, Captain, willing to do me a service, are you? Transport off-world, I need."`
+			choice
+				`	"Where do you want to go?"`
+				`	"I'm not looking for transport jobs right now, sorry."`
+					decline
+			`	"A prestigious research group from Second Cerulean, I represent, and looking to expand our deep-sea expeditions, we are. For that, just settled the acquisition of five new submarines, my superiors have," he says. "To pick them up from <planet>, I was selected, so transported there, I must be. Permitted to pay you <payment> for the service, I am."`
+			choice
+				`	"Alright, I'll show you to my ship."`
+					goto accept
+				`	"Why weren't they built on Second Cerulean in the first place?"`
+				`	"Sorry, I'm not headed that way."`
+					decline
+			`	"No facilities for such construction, Second Cerulean has," he explains. "Remain mostly untouched, the planet must. Build more than simple research stations on it, we cannot. But in need of sturdier subs, we are, so held a competition to determine what company was to build them, my group did. In my opinion, honestly, not needed such a process was. Obvious that House Mallob would win, it was. Unrivaled in hull integrity, they are."`
+			choice
+				`	"I see. Well, I'd love to help you, let me show you to my ship."`
+					goto accept
+				`	"What's so special about this planet that you can't do it there? One shipyard shouldn't be too invasive."`
+				`	"Well, I wish you luck, but I'm not headed that way, sorry."`
+					decline
+			`	He thinks for a while, formulating a response. "Centuries ago, when built there, our outpost was, being worried that trouble the delicate, unique food chains there, further expansion would, prohibited all from settling, and us from constructing any more than the necessary structures, the government did. Enough data, we've gathered, that managed to convince them to allow us some extra exploration vessels, we have. More invasive, the studies with them will be, but enough about the fauna and flora, we now know, that avoid any major complications, we can."`
+			choice
+				`	"Seems like a lot of bureaucracy just for some research. Anyway, I'd be glad to help, let's head to my ship."`
+					goto accept
+				`	"If the planet's native life is all underwater, how would a shipyard pose any risk? It's not like you'd build it underwater."`
+				`	"That sounds overly complicated. I wish you luck in getting it all set, but I'm not headed there, so I can't help you."`
+					decline
+			`	Like a bush in the wind, his body shivers wildy at what you said. "No, built underwater, nothing would be. Obvious that is. But, still, make the rules, I do not, Captain, simply following them I am."`
```

Comment:
```suggestion
			`	Like a bush in the wind, his body shivers wildly at what you said. "No, built underwater, nothing would be. Obvious that is. But, still, make the rules, I do not, Captain. Simply following them, I am."`
```
Personal preference. Just the spelling correction below.
```suggestion
			`	Like a bush in the wind, his body shivers wildly at what you said. "No, built underwater, nothing would be. Obvious that is. But, still, make the rules, I do not, Captain, simply following them I am."`
```
Also, the shiver in the wind reminds me of that one reaction in animal crossing. Just a weird observation